openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
```

Comment:
On a related note, what is the purpose of this method anyway? Seems like a `Lattice` with a `distribcell` tally would accomplish the same thing with much less code, but perhaps I'm not following your logic for implementing this method.